app/messages.py

```diff
@@ -367,7 +329,9 @@
     "message": "An unexpected server error occurs while processing your request. Please try again later."
 }
 
-UNEXPECTED_INPUT = {"message": "Unexpected input is detected. Please check your input to make sure only approved fields are to be submitted."}
+UNEXPECTED_INPUT = {
+    "message": "Unexpected input is detected. Please check your input to make sure only approved fields are to be submitted."
```

Comment:
This should be corrected to `only approved fields are submitted`.

Reply:
This actually was not part of this issue, but it became silently :joy: